### PR TITLE
Request Plugin Id for SRI missing integrity attribute Scanner

### PR DESCRIPTION
**Pull request by cotonne**
Requesting Plugin Id for a new scanner in zap-extension.
Proposition after some talks on the owasp slack/project-zap channel.

### CODE REVIEW

File: docs/scanners.md [hunk 1]
```diff
@@ -177,6 +177,7 @@ Scanners:
 
 90001   Insecure JSF ViewState
 90002   Java Serialized Object
+90003   SRI integrity missing attribute
```
Comment (thc202) [inline docs/scanners.md]: The `missing` should be at the end?

Comment (kingthorin) [reply docs/scanners.md]: Good catch. Also the `I` in the accronym is technically `Integrity` which is redundant to include twice so how about either:
`SRI Attribute Missing`
or
`Subresource Integrity Attribute Missing`

Comment (thc202) [reply docs/scanners.md]: Last one looks good to me.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**